cmd/callrpc.go

```diff
@@ -0,0 +1,58 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/ava-labs/avash/cfg"
+	pmgr "github.com/ava-labs/avash/processmgr"
+	"github.com/spf13/cobra"
+	"github.com/ybbus/jsonrpc"
+)
+
+// CallRPCCmd issues an RPC to a node endpoint using JSONRPC protocol
+var CallRPCCmd = &cobra.Command{
+	Use:     "callrpc [node name] [endpoint] [method] [JSON params] [var scope] [var name]",
+	Short:   "Issues an RPC to a node.",
+	Long:    `Issues an RPC to a node endpoint for the specified method and params.
+	Response is saved to the local varstore.`,
+	Example: `callrpc n1 ext/avm avm.getBalance {"address":"X-KqpU28P2ipUxfTfwaT847wWxyXB4XuWad","assetID":"AVA"} s v`,
```

Comment:
update example for a realistic endpoint. The endpoint for the X-Chain (the AVM) is /ext/bc/X